### PR TITLE
[plugin.video.skyvideoitalia] 1.0.1

### PR DESCRIPTION
**Pull request by nixxo**
### Description

Plugin update:
- added missing logos (removed unused)
- added getSettingAsInt
- added LOCAL_MAP
- improved simplecache usage (custom time set for different use)
- improved cleanTitle (clean if it starts with playlist title)
- better notification ID > NAME
- improved media url check
- moved libraries from resources to libs
- ~~added .gitignore and~~ removed garbage

### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.skyvideoitalia/.gitignore [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# Byte-compiled / optimized / DLL files
```
Comment (enen92) [inline plugin.video.skyvideoitalia/.gitignore]: Please don't submit the gitignore as part of the submission

**Review by enen92 — CHANGES_REQUESTED** [review]
Code looks fine, gitignore has to go. Thanks

**Comment by nixxo** [comment]
@enen92

File: plugin.video.skyvideoitalia/libs/skyitalia.py [hunk 2]
```diff
@@ -208,11 +219,12 @@ def getVideo(self, asset_id):
...
 
         url = None
-        self.log('getPlaylistContent, quality_selected = %s' % self.QUALITIES[int(self.QUALITY)])
+        self.log('getPlaylistContent, quality_selected = %s' % self.QUALITIES[self.QUALITY])
         for i in range(int(self.QUALITY), 0, -1):
             if self.QUALITIES[i] in data:
                 self.log('getPlaylistContent, quality_found = %s' % self.QUALITIES[i])
                 url = data[self.QUALITIES[i]]
-                break
+                if url != '':
```
Comment (enen92) [inline plugin.video.skyvideoitalia/libs/skyitalia.py:227]: (suggestion) if url:

**Review by enen92 — APPROVED** [review]

**Comment by enen92** [comment]
Sorry for the time taken, work has been quite busy lately.

**Comment by nixxo** [comment]
no problem. Thanks for your time.

**Pull request merged**